Smalltalk.gitignore

```diff
@@ -1,15 +1,24 @@
 # changes file
 *.changes
+*.chg
```

Comment:
Same thing with these extensions - can you tell me about what they represent and why they can be safely ignored? Additional resources are definitely helpful here!